<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolFormDialog</class>
 <widget class="QDialog" name="ToolFormDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>サンプル</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>パラメータ</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>ポイントレイヤ選択</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QgsMapLayerComboBox" name="mMapLayerComboBox_selectPoint"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>ファイル選択</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLineEdit" name="lineEdit_file"/>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_file">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>フォルダ選択</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLineEdit" name="lineEdit_directory"/>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_directory">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QgsFieldComboBox" name="mFieldComboBox"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>153</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>ログ</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTextEdit" name="textEdit_log"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_logSave">
             <property name="toolTip">
              <string>ログを保存する</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_logCopy">
             <property name="toolTip">
              <string>ログをクリップボードにコピーする</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_logClear">
             <property name="toolTip">
              <string>ログをクリアする</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS UI Gothic'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;このプラグインについて&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;このプラグインは、自分がプラグインを作成する際に使うソースをまとめたものです。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_cancel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>キャンセル</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_exec">
         <property name="text">
          <string>実行</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_close">
         <property name="text">
          <string>閉じる</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_exec</sender>
   <signal>clicked()</signal>
   <receiver>ToolFormDialog</receiver>
   <slot>click_pushButton_exec()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>ToolFormDialog</receiver>
   <slot>click_pushButton_close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>ToolFormDialog</receiver>
   <slot>click_pushButton_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_logSave</sender>
   <signal>clicked()</signal>
   <receiver>ToolFormDialog</receiver>
   <slot>click_toolButton_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_logCopy</sender>
   <signal>clicked()</signal>
   <receiver>ToolFormDialog</receiver>
   <slot>click_toolButton_copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_logClear</sender>
   <signal>clicked()</signal>
   <receiver>ToolFormDialog</receiver>
   <slot>click_toolButton_clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_file</sender>
   <signal>clicked()</signal>
   <receiver>ToolFormDialog</receiver>
   <slot>click_toolButton_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_directory</sender>
   <signal>clicked()</signal>
   <receiver>ToolFormDialog</receiver>
   <slot>click_toolButton_directory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>click_pushButton_exec()</slot>
  <slot>click_pushButton_close()</slot>
  <slot>click_pushButton_cancel()</slot>
  <slot>click_toolButton_save()</slot>
  <slot>click_toolButton_copy()</slot>
  <slot>click_toolButton_clear()</slot>
  <slot>click_toolButton_file()</slot>
  <slot>click_toolButton_directory()</slot>
 </slots>
</ui>
